src/main/cpp/commands/HickeyEngage.cpp

```diff
@@ -0,0 +1,22 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/HickeyEngage.h"
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.
+// For more information, see:
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+HickeyEngage::HickeyEngage() {
+  SetName("HickeyEngage");
+  AddRequirements(mDooHickey);
```

Comment:
When `mDooHickey` is a reference, change this too

```suggestion
  AddRequirements(&mDooHickey);
```